salt/states/hipchat.py

```diff
@@ -45,7 +45,7 @@ def send_message(name,
                  api_url=None,
                  api_key=None,
                  api_version=None,
-                 message_color='yellow',
+                 color='yellow',
```

Comment:
This should not be changed.  We do not like changing the actual code in patch releases.

The documentation should just be updated.

Reply:
Ah, yeah. my bad! Will revert!